modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,117 @@
+= Topics as Iceberg Tables
+:description: Learn how to integrate Redpanda topics with Iceberg.
+:page-context-links: [{"name": "Linux", "to": "manage:topic-iceberg-integration.adoc" } ]
+:page-categories: Management, High Availability
+
+
+The Redpanda integration with Apache Iceberg allows your topic data to be stored in the cloud in Iceberg's table format. This opens up immediate access to your streaming data for analytical systems such as data warehouses like RedShift, Snowflake, and Clickhouse, and big data processing platforms such as Apache Spark and Flink, without setting up and maintaining additional ETL pipelines. 
+
+== Prerequisites
+
+include::shared:partial$enterprise-license.adoc[]
+
+. To check if you already have a license key applied to your cluster:
++
+[,bash]
+----
+rpk cluster license info
+----
+
+. Install `rpk`.
+
+== Limitations
```

Comment:
Is REST API support for the catalog available in beta?

Reply:
Yes, it is -- but just to be clear, we don't PROVIDE a catalog, we WRITE to another vendor's catalog.  The way Iceberg works is that you create all these data files, and some metadata files, and then you tell a Catalog about them.  We'll be creating the data files out of Kafka topic contents, and we'll generate the relevant metadata, but then we'll call some datalake or other system's catalog by using the REST API and effectively say to that system (like a Snowflake, or Databricks, or whatever), "here is all this new content, add these tables to your datalake"

Reply:
Actually let me amend that.  We don't provide a REST-based catalog.  We CAN generate a pure filesystem-based catalog, which some analytic tools can probably make use of.  But the more sophisticated systems, the data lakes, they HAVE their own catalog.  And in those cases, since the data tables are creating are meant to be additions to those existing datalakes, we end up adding our tables and metadata to their systems by adding/creating entries in THEIR system's catalog.